src/InterfaceUtil.js

```diff
@@ -0,0 +1,18 @@
+import { Console } from "@woowacourse/mission-utils";
+import MESSAGES from "./constants/messages.js";
+
+const InterfaceUtil = {
+  async inputString() {
+    try {
+      return await Console.readLineAsync(MESSAGES.INPUT);
+    } catch (error) {
+      throw new Error(MESSAGES.ERROR);
+    }
+  },
```

Comment:
```suggestion
  inputString() {
    try {
      return Console.readLineAsync(MESSAGES.INPUT);
    } catch (error) {
      throw new Error(MESSAGES.ERROR);
    }
  },
```
여기는 추가적인 로직이 없어서 `async/await`을 제거해도 똑같이 동작할 것 같은데 지예님은 어떻게 생각하시나요??

Reply:
그렇게 해도 같은 동작을 하긴 하겠지만, 뭔가 제가 구현하면서 헷갈릴 것 같아요 ㅎㅎ 어쨌든 `inputString`안의 `readLineAsync`가 비동기니까, `inputString`을 외부에서 사용할 때 '얘는 비동기야!' 라고 말해주는 수단이 될 수 있을 것 같아요. 모르고 놓쳐버릴 것 같은..? 하핳 
이게 제가 `async/await`을 사용한 올바른 이유가 될 수 있을지는 모르겠네요 ㅠㅠ

Reply:
`inputString` 메서드에 `async/await`를 사용해도 사용하는 외부에서는 모르고, 여기 메서드를 봐야 보이는 것 똑같다고 생각해서 메서드 이름에 `inputStringAsync`같이 추가해서 외부에서 사용할 때 비동기 함수라는걸 알려줘도 좋은 것 같네요👍👍👍

Reply:
오 그렇네요 그게 더 좋아보입니다 ㅎㅎ 좋은 의견 너무 감사드려요! ☺️